### DISCUSSION TITLE
Multi value in Filter

### DISCUSSION THREAD

**Discussion by thotam** [rappasoft/laravel-livewire-tables#292] [Feature Requests]
Sorry for my poor English
I want to add filter function, can filter multiple values at once. For example, I want to filter users who have one of the roles Admin, Editor, Publisher

Multiplie in select tag

**Comment by rappasoft** [comment]
Right now I only have support for single select boxes.

**Comment by LaMance22** [comment]
@rappasoft thanks you for your work on this project. Do you plan to add a multi-select feature with (e.g. select2 or Multi Select from your project referrer "WireUI" - https://livewire-wireui.com/docs/select ? I know that currently I can use "MultiSelectFilter" but the problem occurs with more results. I want the user to be able to search in the list and then choose what he needs. Best regards

**Comment by lrljoe** [comment]
Multi Select filters is supported in both drop-down and checkbox form in v2
